Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -224,6 +224,7 @@ namespace RedirectOutput
 
 namespace O3DE::ProjectManager
 {
+    using namespace pybind11::literals;
```

Comment:
None of the pybind11 literals are actually being used.
Even so if they were being used, this should be put into the function which used it instead of the O3DE::ProjectManager namespace